packages/features/ee/workflows/lib/reminders/aiPhoneCallManager.ts

```diff
@@ -20,19 +20,26 @@ type timeUnitLowerCase = "day" | "hour" | "minute";
 function extractPhoneNumber(responses: BookingInfo["responses"]): string | undefined {
   if (!responses) return undefined;
 
-  // Priority 1: CAL_AI_AGENT_PHONE_NUMBER_FIELD first
   const aiAgentPhoneResponse = responses[CAL_AI_AGENT_PHONE_NUMBER_FIELD];
+  
+  // Handle plain string format (how booking form actually stores it)
+  if (typeof aiAgentPhoneResponse === "string") {
+    return aiAgentPhoneResponse;
+  }
+  
+  // Handle object with value property (legacy/alternative format)
```

Comment:
Responses is the format { value, label}. Could you please attach a before/after video where your fix works?